### ISSUE TITLE
Tawk.to Integration

### ISSUE THREAD

**Issue by GaspardC** [BuilderIO/partytown#46]
Hello I tried to integrated the tawk.to widget on my NextJS app.

the service worker is appearing correctly but the widget do not show up and I see some error in the dev tool : "SyntaxError: Unexpected token ')'"
<img width="957" alt="Screenshot 2022-01-12 at 15 14 35" src="https://user-images.githubusercontent.com/9252579/149156773-39d49550-dff5-4801-8cf7-c6c22fd3fb0d.png">

I tried to forward multiple Tawk strings in the Partytown component but without success.

What am I missing ? 
Thanks for your help.

 
<img width="683" alt="Screenshot 2022-01-12 at 15 13 11" src="https://user-images.githubusercontent.com/9252579/149156514-d971d75e-3666-4da0-8a77-cefad88c3cc0.png">

<img width="210" alt="Screenshot 2022-01-12 at 15 14 53" src="https://user-images.githubusercontent.com/9252579/149156768-17813ad4-f1e7-4aa4-8c8f-d21480306d5e.png">
<img width="274" alt="Screenshot 2022-01-12 at 15 17 00" src="https://user-images.githubusercontent.com/9252579/149157066-1db11a71-f0f1-44b7-a74b-62a73aafb215.png">

**Comment by migliori** [comment]
Hello,

I tried Partytown with Tawk.to too, but without success.

**Here's the tawk.to script with Partytown:**
```<script type="text/partytown">
    // xxxx is my id, can't share it here
    var Tawk_API = Tawk_API || {},
        Tawk_LoadStart = new Date();
    (function() {
        var s1 = document.createElement("script"),
            s0 = document.getElementsByTagName("script")[0];
        s1.async = true;
        s1.src = 'https://embed.tawk.to/xxxxxx/default';
        s1.charset = 'UTF-8';
        s1.setAttribute('crossorigin', '*');
        s0.parentNode.insertBefore(s1, s0);
    })();
</script>
```

**And here's the browser console (with Partytown debug enabled)**

![image](https://user-images.githubusercontent.com/7558100/160673554-93117df7-27d7-4e86-bc8d-61faef1cd033.png)

Thanks for your help

**Comment by jesse-deboer** [comment]
Hi! This may be because of a variable '$this' in the script which is being replaced by Partytown.

Could you try it with the version from my PR and see if that helps? https://github.com/BuilderIO/partytown/pull/134

**Comment by migliori** [comment]
@jesse-deboer Thanks for trying to help.

It didn't work, but I searched in twk-chunk-common.js to find out what code was involved in the error and found that the problem seems to come from the "navigator", which is not recognized:

`for(e=0,t=navigator.plugins.length;e<t;e++) {...}`

**Comment by steve8708** [comment]
hey @GaspardC  - we've just created an issue template, could you update your issue here to provide all the information the template asks for? it'll help us look into your issue https://github.com/BuilderIO/partytown/issues/new?assignees=&labels=&template=bug_report.md&title=

pasting below to make things easier:

---

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Reproduction link**
Please include a link to a Stackblitz or Codesandbox reproducing the issue. We will need to see the issue reproduced with hand-written code - we can't debug giant minified third party scripts directly. If you do not include a clean and simple reproduction of your issue, we won't be able to look into it until you do.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Partytown version**
What version of Partytown you are using, e.g. `0.0.1`

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

**Comment by dragansmlinus** [comment]
@steve8708 would you be able to provide a simples example how to integrate partytown and GTM to nextjs app. So far, I'm getting a plenty of CORS issues: `Access to fetch at '<URL>' from origin '<URL>' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

**Comment by steve8708** [comment]
@dragansmlinus for CORS issues see this: https://partytown.builder.io/proxying-requests

**Comment by adamdbradley** [comment]
Closing for now as this issue may be dated. Please create a new issue with a repo case for any specific issues. Thanks

**Comment by ZeldOcarina** [comment]
Hi! This is still not working today! Do I really need to create a new issue?

**Comment by tagperfect** [comment]
> Hello,
> 
> I tried Partytown with Tawk.to too, but without success.
> 
> **Here's the tawk.to script with Partytown:**
> 
> ```
>     // xxxx is my id, can't share it here
>     var Tawk_API = Tawk_API || {},
>         Tawk_LoadStart = new Date();
>     (function() {
>         var s1 = document.createElement("script"),
>             s0 = document.getElementsByTagName("script")[0];
>         s1.async = true;
>         s1.src = 'https://embed.tawk.to/xxxxxx/default';
>         s1.charset = 'UTF-8';
>         s1.setAttribute('crossorigin', '*');
>         s0.parentNode.insertBefore(s1, s0);
>     })();
> </script>
> ```
> 
> **And here's the browser console (with Partytown debug enabled)**
> 
> ![image](https://user-images.githubusercontent.com/7558100/160673554-93117df7-27d7-4e86-bc8d-61faef1cd033.png)
> 
> Thanks for your help

Same here, tried different approaches and this helped me to get rid of this error:

`<script type="text/partytown">
      window.navigator.plugins = window.navigator.plugins || [];
    </script>`

But anyway, chat is not loading
